app/src/main/java/org/thunderdog/challegram/ui/ProfileController.java

```diff
@@ -1873,6 +1873,11 @@ protected void setValuedSetting (ListItem item, SettingView view, boolean isUpda
             }
             break;
           }
+          case R.id.btn_reactions: {
+            int num = chat.availableReactions.length;
+            view.setData(Lang.plural(R.string.xPermissions, num, TdConstants.CHAT_REACTIONS_COUNT));
```

Comment:
Reference to non-existent constant. Kindly update the PR, if you wish the rest of the work to be evaluated by judges and published at https://t.me/tgx_prs.